kobuki_driver/src/tools/simple_keyop.cpp

```diff
@@ -257,6 +299,45 @@ void KobukiManager::processKeyboardInput(char c)
    * the last one for its actual purpose (e.g. left arrow corresponds to
    * esc-[-D) we can keep the parsing simple.
    */
+#if defined(_WIN32)
```

Comment:
Another case of duplicated code; let's try to not have a win32 only codepath

Reply:
added `unordered_map` for virtual key code remapping